test/parallel/test-whatwg-url-custom-tostringtag.js

```diff
@@ -23,10 +23,12 @@ const test = [
   [Object.getPrototypeOf(spIterator), 'URLSearchParams Iterator'],
 ];
 
-test.forEach(([obj, expected]) => {
+
+
+for (const [obj, expected] of test) {
```

Comment:
No need for more empty lines:

```suggestion
for (const [obj, expected] of test) {
```